src/IssueViz.Security/HotspotsList/ViewModels/SecurityCategoryDisplayNames.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.ViewModels
+{
+    /// <summary>
+    /// Based on https://github.com/SonarSource/sonarqube/blob/master/server/sonar-web/src/main/js/helpers/standards.json#L3622
+    /// </summary>
+    internal static class SecurityCategoryDisplayNames
+    {
+        public static readonly IDictionary<string, string> Mapping = new Dictionary<string, string>
```

Comment:
Seems odd for this to be a public dictionary with the lookup/conversion logic somewhere else. Why not just have a public static method and hide the dictionary?

Reply:
Hmm yep. I wanted to minimize the code of this class, so it would be clear that it's a copy-paste. 
I'll add the method.